frontend/packages/dev-console/integration-tests/features/e2e/helm-e2e.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Helm Chart E2E 
+    As a user, I should be able to perform end to end scenarios related to Helm Chart
+
+
+Background:
+    Given user is at developer perspective
+    And user has selected namespace "aut-helm-e2e"
+
+
+@e2e
+Scenario: Helm E2E scenarios
```

Comment:
Update the Scenario name as "Upgrade and uninstall the helm chart"

Reply:
But it's the E2E scenario for Helm chart. If we change the scenario title to "Upgrade and uninstall the helm chart" then it will be related to only upgrade and uninstall helm chart. I think the current title is more descriptive and we should keep the same.

Reply:
Folder description is saying that scenarios belongs to e2e, IMO, scenario description doesn't need E2E word.

Reply:
i will update it.